index.html

```diff
@@ -0,0 +1,341 @@
+				the ocean.</p>
+
+			<p>“SPECTACULAR REALIZATION,” Dr. Guth wrote across the top of the page and drew a double box around it.</p>
+
+			<p>On Monday, Dr. Guth’s starship came in. Radio astronomers reported that they had seen the beginning of
+				the Big Bang, and that his hypothesis, known undramatically as inflation, looked right.</p>
+
+			<p>Reaching back across 13.8 billion years <a
+					href="https://www.youtube.com/watch?v=ZJYc9YmKIO8&amp;feature=youtu.be">to the first sliver of
+					cosmic time</a> with telescopes at the South Pole, a team of astronomers led by John M. Kovac of the
+				Harvard-Smithsonian Center for Astrophysics detected ripples in the fabric of space-time — so-called <a
+					href="http://www.nytimes.com/2016/02/12/science/ligo-gravitational-waves-black-holes-einstein.html?module=inline">gravitational
+					waves</a> — the signature of a universe being wrenched violently apart when it was roughly a
+				trillionth of a trillionth of a trillionth of a second old. They are the long-sought smoking-gun
+				evidence of inflation, proof, Dr. Kovac and his colleagues say, that Dr. Guth was correct.</p>
+
+			<p>Inflation has been the workhorse of cosmology for 35 years, though many, including Dr. Guth, wondered
+				whether it could ever be proved.</p>
+
+			<p>If corroborated, Dr. Kovac’s work will stand as a landmark in science comparable to the recent discovery
+				of dark energy pushing the universe apart, or of the Big Bang itself. It would open vast realms of time
+				and space and energy to science and speculation.</p>
+
+		</div>
+
+		<div id="aside-one">
+			<h5>Editors' Picks</h5>
+			<div class="aside-one-row">
+				<img src="images/A1.png" alt="space">
+				<a
+					href="https://www.nytimes.com/2019/01/07/world/asia/philippines-manila-jail-overcrowding.html?fallback=0&recId=1G4SPPsUZko1MRIdTiNirqBx4mO&locked=0&geoContinent=NA&geoRegion=AB&recAlloc=story-desks&geoCountry=CA&blockId=signature-journalism-vi&imp_id=169928447">Where
+					518 Inmates Sleep in Space for 170, and Gangs Hold It Together</a>
+			</div>
+			<div class="aside-one-row">
+				<img src="images/a2.png" alt="pizza">
+				<a
+					href="https://www.nytimes.com/2019/01/16/dining/slice-joint-pizza-new-york-city.html?fallback=0&recId=1G4SPPsUZko1MRIdTiNirqBx4mO&locked=0&geoContinent=NA&geoRegion=AB&recAlloc=story-desks&geoCountry=CA&blockId=signature-journalism-vi&imp_id=797101841">How
+					the Slice Joint Made Pizza the Perfect New York City Food</a>
+			</div>
+			<div class="aside-one-row">
+				<img src="images/a3.png" alt="church">
+				<a
+					href="https://www.nytimes.com/2019/01/11/nyregion/new-utrecht-reformed-church.html?fallback=0&recId=1G4SPPsUZko1MRIdTiNirqBx4mO&locked=0&geoContinent=NA&geoRegion=AB&recAlloc=story-desks&geoCountry=CA&blockId=signature-journalism-vi&imp_id=673155223">Can
+					a Church Founded in 1677 Survive the 21st Century?</a>
+			</div>
+		</div>
```

Comment:
The editor's pick section does not display properly
Use aside tag
Make sure images are well displayed

Reply:
Hi, @lily-coder ✋ I think when suggesting changes to the code, it’s good to write not only what you want to be corrected, but also **why** you think so. It is about the simple argumentation of your position. Also please make sure to provide links for suggested changes. For example, you can say here:

The editor pick section is not displaying as the original design, Please remember to follow the original design of the article as per project requirements.  You can use the aside tag here in order for better results.  You can check this [article ](https://www.w3schools.com/tags/tag_aside.asp)for more information 👍